platform/podjs/src/browserPod.ts

```diff
@@ -47,7 +47,10 @@ async function openDatabase(): Promise<IDBDatabase> {
                 const keyPath = QUAD_KEYS.filter((_, idx) => (mask >> idx) & 1);
                 polyInStore.createIndex(keyPath.join("-"), keyPath);
             }
-            db.createObjectStore(OBJECT_STORE_POLY_OUT);
+            const polyOutStore = db.createObjectStore(OBJECT_STORE_POLY_OUT, {
```

Comment:
Again, not in the original PR, new behavior. [Here](https://github.com/polypoly-eu/polyPod/pull/810/files#r858230801) you said that this would be called in case the DB was created or the version was changed. Fair enough, but *now* we do seem to have a version change yet the version has not been changed as far as I can tell?
Also, I requested [here](https://github.com/polypoly-eu/polyPod/pull/810/files#r858283617) to document the layout of the database. Without documentation and/or tests, it's impossible to know what kind of behavior is expected; in this case, what kind of indices should be expected in the database. Again, if you need some help in this aspect we can peer on this next Monday or at your earliest convenience.

Reply:
Yes, this change is a breaking change. I didn't bother to advance the database version and implement migration logic since #847 didn't either when dropping the `keyPath` and wrapping files into arrays. It's a bit ironic how I get hold to a higher standards when just trying to make it right than when the original PR was reviewed.

Reply:
Well, they are *your* standards. If we are not going to use them, we might as well drop them.